react/Viewer/ViewerWrapper.jsx

```diff
@@ -4,21 +4,15 @@ import cx from 'classnames'
 
 import styles from './styles.styl'
 
-const ViewerWrapper = ({ className, children, dark }) => (
-  <div
-    className={cx(styles['viewer-wrapper'], className, {
-      [styles['viewer-wrapper--light']]: !dark
-    })}
-    role="viewer"
-  >
+const ViewerWrapper = ({ className, children }) => (
+  <div className={cx(styles['viewer-wrapper'], className)} role="viewer">
```

Comment:
It does not seem that viewer is a proper aria role.

Reply:
hmmm ok je vais investiguer sur les rôles du Viewer et de ses children pour les PR à avenir sur le sujet